### DISCUSSION TITLE
Can u provide to english version?

### DISCUSSION THREAD

**Discussion by lanzzerot** [divertimentos/Curso-Python-Gustavo-Guanabara#3] [Ideas]
The english is global lenguaje, if u wnat to teach a other persons, translate the info to english

**Comment by lanzzerot** [comment]
Use a google translate

**Reply by lanzzerot** [reply]
> Use a google translate

Or chatgpt for translate

**Comment by ggteixeira** [comment]
Although the way you put it is a little bit passive-agressive, nevertheless is a good idea. I'll think about it. Feel free to contribute also by providing some PRs with translations.
